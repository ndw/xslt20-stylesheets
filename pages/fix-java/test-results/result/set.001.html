<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for set.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/base/css/default.css" /><script type="text/javascript" src="../../resources/base/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="../../resources/base/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/base/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output:
          set.001.xml</h1><p>No changes.</p><pre class="show-diff"><span class="tag nodiff">&lt;html&gt;
   <span class="tag">&lt;head/&gt;</span>
   <span class="tag">&lt;body&gt;
      <span class="tag">&lt;article <span class="att">class="set"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag">&lt;header <span class="att">class="set-titlepage"</span>&gt;
            <span class="tag">&lt;h1&gt;Unit Test: set.001&lt;/h1&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Norman&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Walsh&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p <span class="att">class="CVS"</span>&gt;$Id: set.001.xml 4056 2004-11-23 13:28:50Z nwalsh $&lt;/p&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag">&lt;article <span class="att">class="book"</span> <span class="att">id="R.1.2"</span>&gt;
            <span class="tag">&lt;header <span class="att">class="book-titlepage"</span>&gt;
               <span class="tag">&lt;h1&gt;First Book&lt;/h1&gt;</span>
            &lt;/header&gt;</span>
            <span class="tag">&lt;hr/&gt;</span>
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.2.2"</span>&gt;1. Chapter&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;article <span class="att">class="chapter"</span> <span class="att">id="R.1.2.2"</span>&gt;
               <span class="tag">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Chapter 1. Chapter&lt;/h2&gt;</span>
               &lt;/header&gt;</span>
               <span class="tag">&lt;p&gt;Some test data&lt;/p&gt;</span>
            &lt;/article&gt;</span>
         &lt;/article&gt;</span>
         <span class="tag">&lt;article <span class="att">class="book"</span> <span class="att">id="R.1.3"</span>&gt;
            <span class="tag">&lt;header <span class="att">class="book-titlepage"</span>&gt;
               <span class="tag">&lt;h1&gt;Second Book&lt;/h1&gt;</span>
            &lt;/header&gt;</span>
            <span class="tag">&lt;hr/&gt;</span>
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.3.2"</span>&gt;1. Chapter&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;article <span class="att">class="chapter"</span> <span class="att">id="R.1.3.2"</span>&gt;
               <span class="tag">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Chapter 1. Chapter&lt;/h2&gt;</span>
               &lt;/header&gt;</span>
               <span class="tag">&lt;p&gt;Some test data&lt;/p&gt;</span>
            &lt;/article&gt;</span>
         &lt;/article&gt;</span>
         <span class="tag">&lt;article <span class="att">class="book"</span> <span class="att">id="R.1.4"</span>&gt;
            <span class="tag">&lt;header <span class="att">class="book-titlepage"</span>&gt;
               <span class="tag">&lt;h1&gt;Third Book&lt;/h1&gt;</span>
            &lt;/header&gt;</span>
            <span class="tag">&lt;hr/&gt;</span>
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.4.2"</span>&gt;1. Chapter&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;article <span class="att">class="chapter"</span> <span class="att">id="R.1.4.2"</span>&gt;
               <span class="tag">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Chapter 1. Chapter&lt;/h2&gt;</span>
               &lt;/header&gt;</span>
               <span class="tag">&lt;p&gt;Some test data&lt;/p&gt;</span>
            &lt;/article&gt;</span>
         &lt;/article&gt;</span>
      &lt;/article&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre><h2>XML source</h2><pre data-src="../src/set.001.xml"></pre></body><script type="text/javascript" src="../../resources/base/js/prism.js"></script></html>