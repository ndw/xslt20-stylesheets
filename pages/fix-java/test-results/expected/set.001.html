<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: set.001</title><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.2" /><link rel="stylesheet" type="text/css" href="http://localhost:8119/base/css/default.css" /><script type="text/javascript" src="http://localhost:8119/base/js/dbmodnizr.js"></script></head><body><article id="R.1" class="set"><header class="set-titlepage"><h1>Unit Test: set.001</h1><div class="author"><h3><span class="personname"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span></h3></div><p class="CVS">$Id: set.001.xml 4056 2004-11-23 13:28:50Z nwalsh $</p></header><article id="R.1.2" class="book"><header class="book-titlepage"><h1>First Book</h1></header><hr /><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.2.2">1. Chapter</a></span></li></ul></div><article id="R.1.2.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><p>Some test data</p></article></article><article id="R.1.3" class="book"><header class="book-titlepage"><h1>Second Book</h1></header><hr /><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3.2">1. Chapter</a></span></li></ul></div><article id="R.1.3.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><p>Some test data</p></article></article><article id="R.1.4" class="book"><header class="book-titlepage"><h1>Third Book</h1></header><hr /><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.4.2">1. Chapter</a></span></li></ul></div><article id="R.1.4.2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter</h2></header><p>Some test data</p></article></article></article></body></html>